README.md

```diff
@@ -45,6 +49,20 @@ Running:
 docker-compose up
 ```
 
+Restoring the DB from Prod (Linux):
+```bash
+# Enter `password` when prompted for a password after the download step.
+docker-compose exec web script/restore_from_heroku.sh
+```
+
+Restoring the DB from Prod (Mac). The above command will work on Mac, but will take an hour or more due to differences in how docker handles storage. Use the below to reduce how long it takes (will still take a long time).
```

Comment:
I think we'll need to find a way to speed this up but, for now, this gets the job done.